apps/browser/qml/pages/SettingsPage.qml

```diff
@@ -23,11 +23,12 @@ Page {
     id: page
 
     readonly property int _textSwitchIconCenter: Math.round((permissionIcon.width - Theme.itemSizeExtraSmall) / 2.0)
+    readonly property bool homePageBlank: homePageConfig.value === "about:blank"
```

Comment:
Since this is an internal property, it might make sense to also start with an underscore: `_homePageBlank`.